Cargo.lock

```diff
@@ -314,7 +314,7 @@ dependencies = [
 
 [[package]]
 name = "ipc-channel"
-version = "0.6.0"
+version = "0.5.1"
```

Comment:
The revert to 0.5.1 is landed in WR master, so if you rebase all the changes related to ipc-channel should disappear.